### ISSUE TITLE
Error while installing

### ISSUE THREAD

**Issue by pnatsiavas** [yannrivault/packageR_0.1#1]
Using RStudio and R3.5, I get an error while installing queryMed. I am not an R expert and therefore I cannot really investigate further. Could you please advise?
```
> devtools::install_github("yannrivault/queryMed/queryMed")
Downloading GitHub repo yannrivault/queryMed@master
from URL https://api.github.com/repos/yannrivault/queryMed/zipball/master
Installing queryMed
"C:/PROGRA~1/R/R-35~1.0/bin/x64/R" --no-site-file --no-environ --no-save --no-restore  \
  --quiet CMD INSTALL  \
  "C:/Users/pnatsiavas/AppData/Local/Temp/RtmpWOyiL2/devtools2d501c855f58/yannrivault-queryMed-194e99e/queryMed"  \
  --library="C:/Users/pnatsiavas/Documents/R/win-library/3.5" --install-tests 

* installing *source* package 'queryMed' ...
** R
Error in .install_package_code_files(".", instdir) : 
files in 'Collate' field missing from 'C:/Users/pnatsiavas/AppData/Local/Temp/RtmpWOyiL2/devtools2d501c855f58/yannrivault-queryMed-194e99e/queryMed/R':
  data.R
ERROR: unable to collate and parse R files for package 'queryMed'
* removing 'C:/Users/pnatsiavas/Documents/R/win-library/3.5/queryMed'
In R CMD INSTALL
Installation failed: Command failed (1)
```

**Comment by yannrivault** [comment]
@pnatsiavas Thank you for your feedback.
The last updates inserted this error, and we didn't notice it. We are working on it. 
While waiting for corrections, you can install the last release with this line code :
`devtools::install_github("yannrivault/queryMed/queryMed@queryMed_0.5.1")
`
Hoping you can install and use queryMed correctly

**Comment by nolwenn** [comment]
@pnatsiavas The error occured because when updating the package documentation to use roxygen2 one R file had not been commited. Thanks again for reporting the issue.
The problem should be solved if you want to use the devel version of queryMed and call devtools::install_github("yannrivault/queryMed/queryMed").

But as advised by @yannrivault, it might be better (more stable) to use the last released version ( currently 0.5.1)
